__tests__/app.test.js

```diff
@@ -66,49 +66,74 @@ describe("/api/articles", () => {
 				.get("/api/articles/1")
 				.expect(200)
 				.then(({ body }) => {
-					expect(body.articles[0]).toMatchObject({
+					expect(body.articles).toMatchObject({
```

Comment:
If a single article is returned, this should be on a key that reflects the singular, not the plural.